ch07.adoc

```diff
@@ -330,7 +330,7 @@ variables:
     ppn:cell_methods = "time: sum";
   double time(time);
     time:long_name = "time";
-    time:units = "h since 1998-4-19 6:0:0";
+    time:units = "h since 1998-4-19 6:0:0 Z";
```

Comment:
```suggestion
    time:units = "h since 1998-4-19 6:0:0Z";
```

Reply:
I found that the lack of space before `Z` is required: "If the time is in UTCm add a Z directly after the time without a space." (https://en.wikipedia.org/wiki/ISO_8601#Coordinated_Universal_Time_(UTC))